### PR TITLE
fix: home page intro

### LINKED ISSUE #26: Home page video
Field for attaching url need to be added to the Sanity.

Url that need to be added:
https://www.youtube.com/watch?v=J1FJaWuPvbQ&feature=emb_title&ab_channel=SimenAndréWesleyOlsen

### LINKED ISSUE #30: Connect Sanity to Home page
- [x] Update top page content (header, add video url, navigation button);
- [x] Help desk component (title, description, list of services: title, description, link, image, hover-image);
- [ ] Add list of projects (image, url to navigate, name);

### PR DESCRIPTION
**Pull request by TanyaDolgopolova**
- Connect Sanity to a new homepage design
- Write new GraphQl query for getting all data from homepage (without projects section)
- Change design to the new one

Close #26
Close #30

### CODE REVIEW

File: studio/deskStructure.js [hunk 1]
```diff
@@ -27,10 +23,32 @@ export default () =>
...
-        .child(S.documentTypeList('page').title('Page'))
-        .icon(MdLayers),
+        .title('Services Page')
+        .schemaType('servicesPage')
+        .child(
+          S.editor()
+          .id('servicesPage')
+          .schemaType('servicesPage')
+          .documentId('servicesPage'))
+        .icon(MdAssignment),
```
Comment (cobraz) [inline studio/deskStructure.js:33]: Should probably normalize this. «Services Page» seems specific, it should probably be `services` (Services).

Comment (cobraz) [inline studio/deskStructure.js:33]: Should probably normalize this. «Services Page» seems specific, it should probably be `services` (Services).

File: studio/deskStructure.js [hunk 2]
```diff
@@ -27,10 +23,32 @@ export default () =>
...
+      S.listItem()
+        .title('About Page')
+        .schemaType('aboutCompanyPage')
+        .child(
+          S.editor()
+          .id('aboutCompanyPage')
+          .schemaType('aboutCompanyPage')
+          .documentId('aboutCompanyPage'))
+        .icon(MdErrorOutline),
```
Comment (cobraz) [inline studio/deskStructure.js:42]: A golden rule is – «Don't model your content for presentation». Read more about it here: https://www.sanity.io/docs/donts-and-dos

File: studio/deskStructure.js [hunk 3]
```diff
@@ -1,22 +1,18 @@
 import S from '@sanity/desk-tool/structure-builder'
```
Comment (cobraz) [inline studio/deskStructure.js:1]: This pull request could've been split into two. One for changes in `studio/` and one for design changes. It makes it easier to review.

File: web/src/components/Homepage/IntroVideo.tsx [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+        color: 'background',
+        px: '188px',
+      }}
+    >
+      <Box sx={{ my: '32%', width: '60%', zIndex: 2 }}>{children}</Box>
+    </Container>
+  </Fragment>
+)
+
+export default IntroVideo
```
Comment (cobraz) [inline web/src/components/Homepage/IntroVideo.tsx:67]: This whole component appears very obsessive. Considering removing and moving the styles around, and create sub-components to improve readability.

**Review by cobraz — CHANGES_REQUESTED** [review]

File: studio/deskStructure.js (hunk 3 above)
Comment (TanyaDolgopolova) [reply studio/deskStructure.js:1]: Thank you, in the next PRs I will make them separately

**Comment by TanyaDolgopolova** [comment]
@cobraz Checks for branch are not successful, command  `sanity graphql deploy --playground` need to be changed to `sanity graphql deploy --playground --generation gen2`

**Comment by TanyaDolgopolova** [comment]
@cobraz there is some problem with Netlify: `Error: You must login first - run "sanity login"` as I understand we need this plugin https://www.sanity.io/plugins/sanity-plugin-dashboard-widget-netlify

File: studio/deskStructure.js [hunk 5]
```diff
@@ -27,10 +23,32 @@ export default () =>
...
+        .icon(MdErrorOutline),
+      S.listItem()
+        .title('Contacts Page')
+        .schemaType('contact')
+        .child(
+          S.editor()
+          .id('contact')
+          .schemaType('contact')
+          .documentId('contact'))
+        .icon(MdContactMail),
```
Comment (cobraz) [inline studio/deskStructure.js:51]:
```suggestion
        .title('Services Page')
        .schemaType('service')
        .child(
          S.editor()
          .id('service')
          .schemaType('page')
          .documentId('service'))
        .icon(MdAssignment),
      S.listItem()
        .title('About Page')
        .schemaType('about')
        .child(
          S.editor()
          .id('about')
          .schemaType('page')
          .documentId('about'))
        .icon(MdErrorOutline),
      S.listItem()
        .title('Contacts Page')
        .schemaType('contact')
        .child(
          S.editor()
          .id('contact')
          .schemaType('page')
          .documentId('contact'))
        .icon(MdContactMail),
```

«Don't model your content for presentation»: Read more about it here: sanity.io/docs/donts-and-dos

**Review by cobraz — CHANGES_REQUESTED** [review]

**Pull request closed without merging**